source/Outfit.h

```diff
@@ -140,7 +142,8 @@ class Outfit {
 	// Licenses needed to purchase this item.
 	std::vector<std::string> licenses;
 
-	Dictionary attributes;
+	Dictionary<int64_t> rawAttributes;
+	Dictionary<double> cachedAttributes;
```

Comment:
It makes a lot of sense to me to put cachedAttributes on ships realtime in-flight, because we don't want to have the performance overhead of converting int64_t to double when the player is in-flight.
I see less of a need to have this cache directly on outfits. Are there other situations than ships realtime in-flight where it is desired that outfits also have this cache? (If not, then I would expect outfits to only contain the int64_t version of the data.)

Reply:
While we're here, I wonder if it's time for me to finally resurrect #6804. That PR was basically an attribute cache. It was previously done to avoid string comparisons during the attribute lookups, but now it also dodges a division, so it's probably even more beneficial now than it was before.